requirements.txt

```diff
@@ -1,4 +1,5 @@
 requests==2.24.0
 python_telegram_bot==12.8
 beautifulsoup4==4.9.2
-tweepy
\ No newline at end of file
+tweepy
+psycopg2-binary==2.8.6
```

Comment:
What's the use of this `pysycopg2` dependency?

Reply:
Not needed. Removed it.